testproject/tests/conftest.py

```diff
@@ -107,6 +107,19 @@ def active_user_with_twilio_otp() -> UserModel:
     return user
 
 
+@pytest.fixture()
+def active_user_with_twilio_call_otp() -> UserModel:
```

Comment:
Code inside this function and here: `active_user_with_twilio_sms_otp` are the same instead of method_name. maybe extracting it to another function will avoid DRY?